account_reconciliation_widget_due_date/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Reconciliation Widget Due Date",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/account-reconcile",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["account"],
+    "maintainers": ["victoralmau"],
```

Comment:
Putting maintainer and tests you can go to development_status = `Production/Stable`